apps/web-giddh/src/app/tb-pl-bs/components/filter/tb-pl-bs-filter.component.scss

```diff
@@ -56,9 +53,44 @@
     font-size: 18px;
     color: #1a237e !important;
     border: 1px solid $gainsboro;
+}
+.download-data-icon{
+    display: flex;align-items: center;
+    justify-content: flex-end;
+    a{
+        display: flex;
+        width: 32px;
+        height: 32px;
+        justify-content: center;
+        align-items: center;
+        background-color: #fff;
+        font-size: 16px;
+        color: #1a237e !important;
```

Comment:
Use color variable in color and background color

Reply:
Done